proxy/main.go

```diff
@@ -34,19 +34,35 @@ import (
 
 const (
 	restProxyPortEnvVar     = "REST_PROXY_LISTEN_PORT"
+	restProxyGrpcMaxMsgSize = "REST_PROXY_GRPC_MAX_MSG_SIZE"
 	restProxyGrpcPortEnvVar = "REST_PROXY_GRPC_PORT"
 	restProxyTlsEnvVar      = "REST_PROXY_USE_TLS"
 	tlsCertEnvVar           = "MM_TLS_KEY_CERT_PATH"
 	tlsKeyEnvVar            = "MM_TLS_PRIVATE_KEY_PATH"
 )
 
 var (
-	grpcServerEndpoint   = "localhost"
+	grpcServerEndpoint = "localhost"
+	logger             = zap.New()
+
+	// Defaults
 	inferenceServicePort = 8033
-	logger               = zap.New()
 	listenPort           = 8008
+	maxGrpcSize          = 16777216
```

Comment:
nit: I always prefer to include units.. e.g. `maxGrpcMessageSizeBytes`

Reply:
Yea, that's sensible. Changed!